appveyor.yml

```diff
@@ -26,25 +32,30 @@ build_script:
         exit 1
       }
 
-      # Restore packages
+      # Restore Packages
       dotnet restore
 
-      if($env:Framework -eq "net45") {
+      # Set Library Moniker Name
+      if($env:FRAMEWORK -eq "net45") {
         $FrameworkLibsMoniker = "net45"
       } else {
-        $FrameworkLibsMoniker = "netstandard1.6"
+        $FrameworkLibsMoniker = "netstandard1.4"
       }
```

Comment:
I have made one more change in https://github.com/maxmind/GeoIP2-dotnet/pull/68/commits/60bb5bb0195cbf48aea55df4de4985a17a68d6b4; removed this condition and added `FrameworkLibMoniker` as environment variable (for consistency) and changed the following like form `$FrameworkLibMoniker` to `$env:FrameworkLibMoniker`.